### ISSUE TITLE
Use .svg file instead of .png 

### ISSUE THREAD

**Issue by lief-erickson** [briandominick/codewriting#16]
> diagram-docops-general.png
> venn-docs-knowledge-truth.png
> venn-docs-knowledge-truth-proportionate.png

Given that these are vector images, ideally make as an SVG instead of PNG. Should someone decide to translate this document, it would be easier to translate these files, too. Besides, SVG is well supported now.

https://caniuse.com/#search=svg

**Comment by briandominick** [comment]
Yeah, I have no excuse for not using SVG except I think I was maybe cropping in an app that doesn't handle SVG. I'm sourcing in OmniGraffle and now I have Boxy, so no excuses.
